app/views/admin/role_appointments/new.html.erb

```diff
@@ -1,17 +1,36 @@
-<% page_title "Create new appointment to #{@role_appointment.role.name}"  %>
+<% content_for :page_title, "Create new appointment to #{@role_appointment.role.name}"  %>
+<% content_for :title, "Create new appointment to #{@role_appointment.role.name}" %>
+<% content_for :title_margin_bottom, 6 %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @role_appointment, parent_class: "role_appointment")) %>
```

Comment:
same comment as before about not needing parent_class https://github.com/alphagov/whitehall/pull/7507#discussion_r1156799071
```
<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @role_appointment) %>
```

Reply:
✅